internal/xds/translator/translator_test.go

```diff
@@ -48,6 +48,15 @@ func TestTranslate(t *testing.T) {
 		{
 			name: "http-route-request-headers",
 		},
+		{
+			name: "http-route-response-add-headers",
```

Comment:
nit: can the response header test be combined into one (added to the `routes` list)?

Reply:
Thanks, but let us keep this for now, Okay?